src/main/java/com/paneedah/mwc/weapons/Bullets.java

```diff
@@ -331,6 +332,30 @@ public static void init(Object mod) {
                 .withTextureName("Dummy.png")
                 .build(MWC.modContext, ItemBullet.class);
         
+        Bullet277 = new ItemBullet.Builder()
+
+                .withCreativeTab(MWC.AMMUNITION_AND_MAGAZINES_TAB)
+                .withName("Bullet277")
```

Comment:
name is not following MC convetion, it should be `bullet_277`

Reply:
? I was not aware since all the other bullets .withName was like .withName("Bullet9x19mm") unless you're talking about something else

Reply:
Oh okay, I’ll correct this when I can thank you.